SPEC.md

```diff
@@ -217,40 +243,37 @@ Success is indicated by a zero return code and the following JSON being printed
 ```
 {
   "cniVersion": "0.1.0",
```

Comment:
What did we agree on the CNI version bump? Can we implement this backwards compatible?